src/content/docs/infrastructure/host-integrations/host-integrations-list/mongodb/mongodb-migration-guide.mdx

```diff
@@ -0,0 +1,316 @@
+---
+title: MongoDB monitoring migration guide
+tags:
+  - Integrations
+  - On-host integrations
+  - On-host integrations list
+metaDescription: "New Relic's MongoDB integration: how migrate from legacy to to the new version."
+---
+
+The existing legacy version will be deprecated and no longer supported during FY24.
+
+### Why do I need to migrate to the new version?
```

Comment:
do we want to add a callout clarifying the status of kubernetes support?

Reply:
Yes, good catch!

I'm going to add a section in the page.

Reply:
Done!